README.md

```diff
@@ -177,6 +177,31 @@ Call a gRPC endpoint using a JSON input. What this does behind the scenes:
 
 *See [docs/grpc.md](docs/grpc.md) for full instructions.*
 
+### SSL/TLS Connections
+
+To enable SSL/TLS connections to the server, use the `--tls` command line flag. 
+If not specified all the following flags will be ignored.
+
+By default host validation is enabled. To disable, pass the `--insecure` command line flag.
```

Comment:
`--tls --insecure` doesn't make sense to me - having a `--insecure` flag to use as a TLS option seems counterintuitive. Perhaps `--skip-host-validation` or similar, open to suggestions.

Reply:
I actually completely agree.  It was succinct.  Chef uses "--ssl-verify-mode none".  Wget uses "--no-check-certificate".  Curl (which is why the grpcurl library uses):
-k, --insecure

(TLS) By default, every SSL connection curl makes is verified to be secure. This option allows curl to proceed and operate even for server connections otherwise considered insecure.

The server connection is verified by making sure the server's certificate contains the right name and verifies successfully using the cert store.

See this online resource for further details: https://curl.haxx.se/docs/sslcerts.html